environment.yml

```
@@ -1,7 +1,6 @@
-name: sus-env2
+name: sus-env3
 dependencies:
 - ipython
-- python=2
-- ujson
-- pip:
-  - js2py
+- python=3.11
```

```suggestion
- python=3.12
```
Possible? No reason to go out of the gate with an outdated Python